crates/bevy_pbr/src/light_probe/generate.rs

```diff
@@ -0,0 +1,1084 @@
+//! Generated environment map filtering.
+//!
+//! A *generated environment map* converts a single, high-resolution cubemap
+//! into the pair of diffuse and specular cubemaps required by the PBR
+//! shader. Add [`bevy_light::GeneratedEnvironmentMapLight`] to a camera
+//! and Bevy will, each frame, generate the diffuse and specular cubemaps
+//! required by the PBR shader.
+//!
+//! 1. Copy the base mip (level 0) of the source cubemap into an intermediate
```

Comment:
A little too technical, maybe. I think it should focus more on user usage. I try to write rendering docs keeping in mind people who have never heard the word "PBR" before, and are basically learning rendering solely from reading bevy's docs.

You don't really have to explain what an env map is, you can like to EnviornmentMapLight, but it should defenitly focus on how this differs from that and when you would want to use one or the other.

Reply:
resolved by https://github.com/mate-h/bevy/pull/8